core/src/main/java/io/questdb/griffin/engine/table/AbstractDescendingRecordListCursor.java

```diff
@@ -36,6 +36,7 @@ abstract class AbstractDescendingRecordListCursor extends AbstractDataFrameRecor
 
     protected final DirectLongList rows;
     private long index;
+    protected boolean isOpen;
```

Comment:
I'd make it private with read-only get access.
`close()` on this class should set it to false. At the moment it's not obvious what can change `isOpen` and in what order

Reply:
Should be private